pkg/test/test.msbuild

```diff
@@ -4,22 +4,44 @@
     <TestPackages Condition="'$(TestPackages)' != ''" Include="$(TestPackages)" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <PackageTestProjectsDir Condition="'$(PackageTestProjectsDir)' == ''">$(MSBuildThisFileDirectory)..\projects</PackageTestProjectsDir>
+    <_projectProperties>SupportFilesDir=$(MSBuildThisFileDirectory)</_projectProperties>
+
+    <!-- We need this to be able to pass restore sources from command line when running tests in helix -->
+    <_restoreSources Condition="'$(PackageTestSources)' != ''">$([System.String]::Copy('$(PackageTestSources)').Replace('+',';'))</_restoreSources>
```

Comment:
I didn't see where this property was set.  I see you set RestoreSources but not PackageTestSources.